### PR TITLE
upd(opentoonz): `1.5.0` -> `1.6.0`

### PR DESCRIPTION
**Pull request by echometerain**

### CODE REVIEW

**Review by wizard-28 — CHANGES_REQUESTED** [review]
The CI fails

**Comment by wizard-28** [comment]
@echometerain status on this?

**Comment by echometerain** [comment]
@wizard-28 all checks except debian testing is successful
and pacstall is a package manager for ubuntu

**Review by Henryws — CHANGES_REQUESTED** [review]
```
install: target '/usr/src/pacstall/opentoonz/usr/share/applications/' is not a directory: No such file or directory
install: target '/usr/src/pacstall/opentoonz/usr/share/pixmaps/' is not a directory: No such file or directory
```

File: packages/opentoonz/opentoonz.pacscript [hunk 1]
```diff
@@ -38,6 +38,8 @@ build() {
 install() {
 
   cd "toonz/build" || return
+  sudo mkdir -P "/usr/src/pacstall/opentoonz/usr/share/pixmaps/"
```
Comment (Henryws) [inline packages/opentoonz/opentoonz.pacscript:41]: These `mkdir`s should use `-p`. `-P` is not a flag.

**Review by Henryws — CHANGES_REQUESTED** [review]

**Comment by echometerain** [comment]
I have no idea why this doesn't work, it runs on my machine perfectly
image

**Comment by cat-master21** [comment]
It's only in Debian stable, and not Sid. For Ubuntu It's only in Impish and Hirsute: https://packages.ubuntu.com/impish/libgsl25. It is not in Ubuntu Devel (Jammy) or Debain Sid so those checks failed.

**Comment by cat-master21** [comment]
Libgls27 is in unstable Debian repos: https://packages.debian.org/search?keywords=libgsl27

**Comment by cat-master21** [comment]
Try Libgsl-dev, @echometerain https://packages.debian.org/sid/libgsl-dev

**Comment by echometerain** [comment]
thank you @cat-master21! It works now!

File: packages/opentoonz/opentoonz.pacscript [hunk 2]
```diff
@@ -47,6 +47,9 @@ install() {
 
   sudo install -Dm755 "toonz/sources/xdg-data/io.github.OpenToonz.desktop" "${STOWDIR}/${name}/usr/share/applications/"
   sudo install -Dm644 "toonz/sources/xdg-data/io.github.OpenToonz.png" "${STOWDIR}/${name}/usr/share/pixmaps/"
+  
+  print "#\!/bin/sh\nsh /opt/opentoonz/bin/opentoonz" | sudo tee /usr/bin/opentoonz
```
Comment (Henryws) [inline packages/opentoonz/opentoonz.pacscript:51]:
```bash
Error: no such file "#\!/bin/sh\nsh /opt/opentoonz/bin/opentoonz"
```

**Review by Henryws — CHANGES_REQUESTED** [review]

**Review by Henryws — DISMISSED** [review]

**Comment by Henryws** [comment]
Oops misclick

File: packages/opentoonz/opentoonz.pacscript (hunk 2 above)
Comment (echometerain) [reply packages/opentoonz/opentoonz.pacscript:51]: lmao zsh moment

File: packages/opentoonz/opentoonz.pacscript [hunk 3]
```diff
@@ -1,56 +1,61 @@
...
+  sudo chmod +x "/usr/bin/opentoonz"
 }
 
 removescript() {
 
   ask "Do you want to remove your opentoonz config?" N
-  if [[ $answer -eq 1 ]]; then
-    sudo rm -rf "home/$LOGNAME/.config/OpenToonz/"
+  if [[ ${answer} -eq 1 ]]; then
+    sudo rm -rf "home/${LOGNAME}/.config/OpenToonz/"
```
Comment (Henryws) [inline packages/opentoonz/opentoonz.pacscript:59]: Shouldn't this have a `/` in front? Also why `sudo`?

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/opentoonz/opentoonz.pacscript [hunk 4]
```diff
@@ -1,56 +1,61 @@
...
+  cd "../../" || return
+  sudo install -Dm644 LICENSE.txt "${STOWDIR}/${name}/usr/share/licenses/${name}/LICENSE.txt"
+  sudo chmod +x "${STOWDIR}/${name}/opt/${name}/bin/opentoonz"
 
-  sudo install -Dm755 "toonz/sources/xdg-data/io.github.OpenToonz.desktop" "$STOWDIR/$name/usr/share/applications/"
-  sudo install -Dm644 "toonz/sources/xdg-data/io.github.OpenToonz.png" "$STOWDIR/$name/usr/share/pixmaps/"
+  sudo install -Dm755 "toonz/sources/xdg-data/io.github.OpenToonz.desktop" "${STOWDIR}/${name}/usr/share/applications/"
+  sudo install -Dm644 "toonz/sources/xdg-data/io.github.OpenToonz.png" "${STOWDIR}/${name}/usr/share/pixmaps/"
+
+  printf '#!/bin/sh\nsh /opt/opentoonz/bin/opentoonz' | sudo tee /usr/bin/opentoonz
```
Comment (Henryws) [inline packages/opentoonz/opentoonz.pacscript:51]: Redirect `tee` to `/dev/null`

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/opentoonz/opentoonz.pacscript [hunk 5]
```diff
@@ -48,7 +48,7 @@ install() {
   sudo install -Dm755 "toonz/sources/xdg-data/io.github.OpenToonz.desktop" "${STOWDIR}/${name}/usr/share/applications/"
   sudo install -Dm644 "toonz/sources/xdg-data/io.github.OpenToonz.png" "${STOWDIR}/${name}/usr/share/pixmaps/"
 
-  printf '#!/bin/sh\nsh /opt/opentoonz/bin/opentoonz' | sudo tee /usr/bin/opentoonz
+  printf '#!/bin/sh\nsh /opt/opentoonz/bin/opentoonz' | sudo tee "/usr/bin/opentoonz" > "/dev/null"
```
Comment (Henryws) [inline packages/opentoonz/opentoonz.pacscript:51]:
```suggestion
  printf '#!/bin/sh\nsh /opt/opentoonz/bin/opentoonz' | sudo tee /usr/bin/opentoonz > /dev/null
```

**Review by Henryws — CHANGES_REQUESTED** [review]

**Review by Henryws — APPROVED** [review]

**Comment by cat-master21** [comment]
@echometerain use Libgsl-dev: https://packages.ubuntu.com/jammy/libgsl-dev not libgsl25 as it is only in Impish and Hirsuite (Jammy has libgsl27).

**Comment by echometerain** [comment]
> @echometerain use libgsl-dev: https://packages.ubuntu.com/jammy/libgsl-dev not libgsl25 as it is only in Impish and Hirsuite (Jammy has libgsl27).

I already added it

**Comment by cat-master21** [comment]
Whops..Who's... I forgot the comment was from 5 days ago... A bit embarrassing.

**Review by Henryws — APPROVED** [review]

**Review by cat-master21 — APPROVED** [review]

**Pull request merged**